### PR TITLE
Form Validation

### PR DESCRIPTION
**Pull request by MrTaaJ**
This PR contains the following:

- Add some HTML syntax to show while querying the form input,
- Add styling to the new HTML syntax created,
- Query every input in JavaScript to validate the required format and make sure it is not empty

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -120,3 +120,70 @@ popDeactivators.forEach(popDeactivator => {
...
+    document.querySelector('.emailArea-note').classList.remove('textarea-note-active');
+    if (eventBut(textValid) === true || eventBut(nameValid) === true) {
+      defaultButton(true);
+    } else if (textValid.value.length < 15) {
+      defaultButton(true);
+    } else {
+      defaultButton(false);
+    }
+  }
+};
```
Comment (rotimiazeez) [inline js/app.js]: - [ ] Kindly check if the email is all lowercase by comparing if the email and its lowercase converted version are the same, ie: `email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string. Any implementation of this check should be considered valid as long as it works properly😎😎 in simple terms, the contact form contains at least one validation checking if the email is written in all lowercase:

* If a user submits the contact form with the email field containing uppercase characters an error message should show up.
*

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @MrTaaJ 👋, 
## Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- [ ] Just a few more changes left for you to be approved💯😉

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: js/app.js (hunk 1 above)
Comment (MrTaaJ) [reply js/app.js]: Noted boss.
I have made some modifications to the code to help enable that the email is always in lowercase.

**Review by rotimiazeez — APPROVED** [review]
# Hi @MrTaaJ , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 💯🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------

**Pull request merged**